src/utils/AdaptiveTimeStep.cpp

```diff
@@ -305,7 +308,7 @@ AdaptiveTimeStep::GatherMinAccSlice (MultiBeam& beams, const amrex::Geometry& ge
                 doGatherEz(xp, yp, Ezp, slice_arr, ez_comp,
                            dx_inv, dy_inv, x_pos_offset, y_pos_offset);
                 return {
-                    charge_mass_ratio * Ezp * clightinv
+                    -std::abs(Ezp * clightinv)
```

Comment:
This is a bit tricky: in principle, the adaptive time step should work fine whether the accelerating force (q*Ez) is positive (time step gets larger) or negative (time step gets smaller). In practical scenarios, this was satisfied with the current implementation. Do you think this is preserved?